code/modules/spells/spell_types/shapeshift/wolf.dm

```diff
@@ -0,0 +1,58 @@
+/datum/action/cooldown/spell/shapeshift/wolf
+	name = "Wolf Form"
+	desc = "Take on the shape a wolf."
+	invocation = span_danger("<b>%CASTER</b> lets out a mighty growl!")
+	invocation_self_message = span_danger("You let out a mighty growl!")
+	invocation_type = INVOCATION_EMOTE
+	spell_requirements = NONE
+
+	possible_shapes = list(/mob/living/basic/mining/wolf)
+
+/obj/item/clothing/neck/cloak/wolf_coat
+	name = "wolf pelt cloak"
+	desc = "A cloak made of very lively wolf fur, feels warm to touch."
+	icon_state = "icecloak"
+	icon = 'icons/obj/clothing/cloaks.dmi'
+	worn_icon = 'icons/mob/clothing/neck.dmi'
+	inhand_icon_state = "icecloak"
+	lefthand_file = 'icons/mob/inhands/clothing/neck_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/clothing/neck_righthand.dmi'
+	body_parts_covered = CHEST|GROIN|ARMS
+
```

Comment:
You should include the FIRE_PROOF resistance flag on the cloak and maybe the hood. Lavaland items and artifacts share this so they don't evaporate upon immediate contact with their most common hazard, fire stacks.

It doesn't have to provide any protections to the wearer, just the item itself should be protected.

Reply:
It's just wolf skin so I don't think it makes sense to be fireproof, but make it freezeproof.

Reply:
That's not what I'm asking. FIRE_PROOF only matters for the object itself, not the wearer. If it isn't FIRE_PROOF, since it is external clothing, it'll burn away the moment the wearer is lit on fire, even if the wearer is protected from the fire from some other source (like drake armor). 

That's why FIRE_PROOF flag is consistently added to lavaland/icebox clothing items.

Reply:
It's inconsistently added for items made of pelt, bones and sinew.

Bone bracers, sinew skirt, skull codpiece, bone talisman, sinew handcuffs, sinew fishing line reel, bone fishing rod are examples of items WITHOUT fireproof. Winterboots & ice hiking boots also isn't fireproof but those aren't confirmed to be made out of animal fur.

Goliath Cloak, Ash Drake armor, bone axe & wreaths are ones WITH fireproof

I don't give a shit about adding it but there's no "consistency" here to follow.

Reply:
Bone bracers were an oversight on my part. The accessories are a bit mixed and those other items are not clothing, but all could be made consistent.

I am _attempting_ to push for consistency going forward slowly as I make some lavaland adjustments. That's why I care here. Just highlighting the inconsistencies to me doesn't change the fact that I want to bring those items in line as well.

(And you can help if you're so inclined because it will make me happy)